Automattic/jetpack
# Search: Add tooltips to show upgraded usage limits information

Fixes #26739 

#### Changes proposed in this Pull Request:

* Introduce props `shadowAnchor` and `forceShow` to support pure `Popover` displaying within the parent wrapper.
* Add tooltips to `DonutMeter` to inform users about the upgraded usage limits.
* Introduce the feature `just_upgraded` within the URL to show the tooltips temporarily.

#### Other information:


#### Jetpack product discussion


p1HpG7-hEH-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Purchase the Jetpack Search `Free` plan.
* Navigate to the site with URL `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1&features=just_upgraded`.
* Ensure the `Record` tooltip displays above the DonutMeter.

<img width="1198" alt="截圖 2022-10-12 下午11 40 32" src="https://user-images.githubusercontent.com/6869813/195387875-f950d5ad-151b-45b2-b196-c3fc820234b1.png">

* Click the `Next` button on the `Record` tooltip.
* Ensure the `Record` tooltip disappears and the `Request` tooltip displays above the DonutMeter.

<img width="1198" alt="截圖 2022-10-12 下午11 40 37" src="https://user-images.githubusercontent.com/6869813/195388013-3f5343b5-61d8-4ee1-bdb7-d4731151803e.png">

* Click the `Finish` button on the `Request` tooltip.
* Ensure both tooltips not showing anymore.

> Note: The flag to store the read status for users will be implemented as a follow-up PR.
